ros_buildfarm/templates/doc/doc_independent_job.xml.em

```diff
@@ -64,7 +64,7 @@ repo_name = os.path.splitext(os.path.basename(repo_url))[0]
     'builder_shell',
     script='\n'.join([
         'echo "# BEGIN SECTION: Clone %s"' % repo_name,
-        'python3 -u $WORKSPACE/ros_buildfarm/scripts/wrapper/git.py clone --depth 1 %s $WORKSPACE/repositories/%s' % (repo_url, repo_name),
+        'python3 -u $WORKSPACE/ros_buildfarm/scripts/wrapper/git.py clone --recurse-submodules --depth 1 %s $WORKSPACE/repositories/%s' % (repo_url, repo_name),
```

Comment:
This is only used for the folllowing repos and neither of them uses submodules: https://github.com/ros-infrastructure/ros_buildfarm_config/blob/01e30d05362b4763567c459e8e4612f7dc49619b/doc-independent-build.yaml#L5

So this change isn't needed.

Reply:
Local doc jobs are already using the devel template for cloning the repositories, see https://github.com/ros-infrastructure/ros_buildfarm/blob/93d19cb8003ecb34317c63c0ad0284973ddb04c3/ros_buildfarm/templates/doc/doc_script.sh.em#L44